test/clubbing_top_level.txt

```diff
@@ -0,0 +1 @@
+<table border="1"><tr><th>aaa</th><th>ccc</th></tr><tr><td>1</td><td>2</td></tr><tr><td>bbb</td><td>2</td></tr></table>
```

Comment:
Hey Varun. Sorry, was busy for a while with my small child. Did you get my gitter invitation? I thought it's probably easier if we can chat. Otherwise I'll specify here what I mean, I think I understand now the reason behind the confusion earlier.